whats_fresh/whats_fresh_api/templates/product.html

```diff
@@ -96,16 +102,67 @@ <h4 id='error_tag' class='light warning'></h4>
 
 {% endblock content %}
 {% block footer %}
+<script src="//ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.min.js"></script>
+<script src="//ajax.googleapis.com/ajax/libs/jqueryui/1.8.23/jquery-ui.min.js"></script>
+<link href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8.23/themes/smoothness/jquery-ui.css" rel="stylesheet" type="text/css" media="all" />
```

Comment:
Upgrade to the latest version of jquery ui and make sure things work: https://jqueryui.com/ https://developers.google.com/speed/libraries/#jquery-ui